packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -232,6 +237,15 @@ export const contract = async (
     },
   };
 
+  const resolverZone = zone.subZone('CCTPResolver');
+  const resolverKit = makeResolverKit(resolverZone, zcf, vowTools)();
+
+  const cctpResolver = {
+    client: resolverKit.client,
+    service: resolverKit.service,
+  };
+  const makeResolverInvitationMakers: ResolverInvitationMakers = resolverKit.invitationMakers;
```

Comment:
Why not just get the kit facets by deconstruction?
```suggestion
  const {
    client: resolverClient,
    invitationMakers: makeResolverInvitationMakers,
  } = makeResolverKit(resolverZone, zcf, vowTools)();
```